### PR TITLE
Fixing visualisation editor UI bugs

### LINKED ISSUE #104: UI bugs in visualisation editor page
### Description
There are some bugs in the data visualisation editor page.
1. The preview of the visualisation will re-render with new data every time a letter is written in the textarea.
2. Large labels will push the visualisation so it gets smaller.
3. Changing text will mess up the visualisation with medium and small visualisation.

### Steps to reproduce
1. Run the application
2. Navigate to visualisation editor  page.

### Expected behaviour
The bugs described above should not be happening.

### PR DESCRIPTION
**Pull request by emilom**
Made some changes in the visualisation editor/preview components. The mocked graph will now stay the same at each render. You can now add text to a graph without the graph looking like a dubstep track on youtube 🎉 I also made some changes in the dashboard item component. It will now cover more space and I think it looks a bit better. There was also a bug where the width of the preview stayed the same after you added text, which is now removed. 
Fixed some overflow problems with tags and paragraphs. 
Made the axis of the graphs scale with different widths.

I don't think you storybook is the best place to review these changes, so check nellify as well :)

Close #104

### CODE REVIEW

File: src/components/molecules/VisualisationPreview.tsx [hunk 1]
```diff
@@ -32,13 +32,15 @@ const VisualisationPreview: React.FC<VisualisationPreviewProps> = ({ metadata, s
             return () => null;
         }
 
+        const timeEtryMockData = mockTimeEntry(100, visualisation.axes.y.limit);
```
Comment (FredrikAugust) [inline src/components/molecules/VisualisationPreview.tsx]: Typo. Missing `n`

**Review by FredrikAugust — CHANGES_REQUESTED** [review]
A few notes;)

- Entering a very long description breaks the preview
Screenshot 2021-04-06 at 16 20 07

- The graph re-renders every time you edit the description. When the text is pushing the left side of the graph, it will re-render a lot of times while "animating" to it's new size. I'd recommend turning on "paint on rerender" in React dev tools and using the app.

**Comment by emilom** [comment]
> A few notes;)
> 
> * Entering a very long description breaks the preview
>   Screenshot 2021-04-06 at 16 20 07
> * The graph re-renders every time you edit the description. When the text is pushing the left side of the graph, it will re-render a lot of times while "animating" to it's new size. I'd recommend turning on "paint on rerender" in React dev tools and using the app.

Weird.. It works for me with overflow hidden. Tested with chrome and safari. The text should not be pushing the graph.

**Comment by FredrikAugust** [comment]
> > A few notes;)
> > 
> > * Entering a very long description breaks the preview
> >   Screenshot 2021-04-06 at 16 20 07
> > * The graph re-renders every time you edit the description. When the text is pushing the left side of the graph, it will re-render a lot of times while "animating" to it's new size. I'd recommend turning on "paint on rerender" in React dev tools and using the app.
> 
> Weird.. It works for me with overflow hidden. Tested with chrome and safari. The text should not be pushing the graph.

It was tested in Firefox on the deployed site (netlify).

**Comment by emilom** [comment]
> > > A few notes;)
> > > 
> > > * Entering a very long description breaks the preview
> > >   Screenshot 2021-04-06 at 16 20 07
> > > * The graph re-renders every time you edit the description. When the text is pushing the left side of the graph, it will re-render a lot of times while "animating" to it's new size. I'd recommend turning on "paint on rerender" in React dev tools and using the app.
> > 
> > 
> > Weird.. It works for me with overflow hidden. Tested with chrome and safari. The text should not be pushing the graph.
> 
> It was tested in Firefox on the deployed site (netlify).

Turns out you need
`white-space: pre-wrap;`
`word-break: break-all;`

for firefox and IE. Should be fixed now :)

**Review by FredrikAugust — APPROVED** [review]
Works like a charm

**Pull request merged**